chall02/szavarce.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import sys
+codigo_morse = {
+  "A": ".-", "B": "-...", "C": "-.-.", "D": "-..", "E": ".", "F": "..-.", "G": "--.", "H": "....", "I": "..", "J": ".---", "K": "-.-", "L": ".-..", "M": "--", "N": "-.", "O": "---", "P": ".--.", "Q": "--.-", "R": ".-.", "S": "...", "T": "-", "U": "..-", "V": "...-", "W": ".--", "X": "-..-", "Y": "-.--", "Z": "--..", " ": " "
+}
+resultado = ""
+if len(sys.argv) != 2:
+  print("usage: ./szavarce.py <a-zA-Z string>")
```

Comment:
The filename might change, you should make the message dynamic:
```suggestion
  print("usage: %s <a-zA-Z string>" % sys.argv[0])
```